spec/quality_spec.rb

```diff
@@ -73,6 +89,18 @@ def check_for_extra_spaces(filename)
     expect(error_messages.compact).to be_well_formed
   end
 
+  it "maitains language quality of the documentation" do
+    included = /ronn/
+    error_messages = []
+    Dir.chdir(File.expand_path("../../man", __FILE__)) do
```

Comment:
hey, won't it end up being too complicated if we check for all files? Just writing a parser for comments will be too much if we are to do it right (and cover all kind of [possible comments in ruby](http://stackoverflow.com/questions/2989762/multi-line-comments-in-ruby#answer-2991254)).

Can we use rubocop for this or limit the type of comments we allow? Also, by "source" I hope you mean just the `lib` folder.